README.md

```diff
@@ -1,52 +1,66 @@
-e-mission phone app
---------------------
+# [e-mission phone app](https://github.com/e-mission/e-mission-phone/tree/master)
 
-This is the phone component of the e-mission system.
+__This is the phone component of the e-mission system.__
 
-:sparkles: This has now been upgraded to cordova android@9.0.0 and iOS@6.0.1 ([details](https://github.com/e-mission/e-mission-docs/issues/554)). It has also been upgraded to [android API 29](https://github.com/e-mission/e-mission-phone/pull/707/), [cordova-lib@10.0.0 and the most recent node and npm versions](https://github.com/e-mission/e-mission-phone/pull/708)It also now supports CI, so we should not have any build issues in the future. The limitations from the [previous upgrade](https://github.com/e-mission/e-mission-docs/issues/519) have all been resolved. This should be ready to build out of the box, after all the configuration files are changed.
+:sparkles: This has now been upgraded to cordova android@12.0.0 and iOS@6.2.0. It has also been upgraded to the **latest Android & iOS versions**, **cordova-lib@10.0.0 and the most recent node and npm versions**. It also now supports CI, so we should not have any build issues in the future. __This should be ready to build out of the box.__
```

Comment:
Why have the versions not been removed?
https://github.com/e-mission/e-mission-phone/pull/1037#discussion_r1335066508